cmd/krane/go.mod

```diff
@@ -10,49 +10,49 @@ replace (
 
 require (
 	github.com/google/go-containerregistry v0.8.1-0.20220110151055-a61fd0a8e2bb
-	github.com/google/go-containerregistry/pkg/authn/k8schain v0.0.0-20211223213658-2874338840a6
+	github.com/google/go-containerregistry/pkg/authn/k8schain v0.0.0-20220114205711-890d5b362eb8
```

Comment:
I believe you'll need to update this in a separate PR

Reply:
Actually - probably not worth doing this here until k8s chain merges

Reply:
I'm not sure we need to do this in another PR, since it's just `replace`d above. If downstream deps who don't respect the `replace` are broken by this (I think that's what you're describing?), we should figure out a way to capture that in a presubmit to prevent future breakages.

In any case, I can `go get -u` again after this merges. I'm kinda hating all these interdependent modules in this repo 🤦 .

Reply:
> In any case, I can go get -u again after this merges.

Oh nm - I didn't see the `replace` directive. That's not necessary then. It would be If you want `krane` to be `go install`-able. 

>  I'm kinda hating all these interdependent modules in this repo 🤦 .

Yeah the explicit refs are not fun - I'll open a go issue ;)